common.sh

```diff
@@ -207,6 +207,7 @@ export EXTRA_NODES_FILE=${EXTRA_NODES_FILE:-"${WORKING_DIR}/${CLUSTER_NAME}/extr
 NODES_PLATFORM=${NODES_PLATFORM:-"libvirt"}
 BAREMETALHOSTS_FILE=${BAREMETALHOSTS_FILE:-"${OCP_DIR}/baremetalhosts.json"}
 EXTRA_BAREMETALHOSTS_FILE=${EXTRA_BAREMETALHOSTS_FILE:-"${OCP_DIR}/extra_baremetalhosts.json"}
+BMO_WATCH_ALL_NAMESPACES="${BMO_WATCH_ALL_NAMESPACES:-false}"
```

Comment:
quotes (") not needed except for around false, that should be "false", to be consistent with other declarations
also you probably want to export the variable

Reply:
Added the quotes because that's what `shellcheck` recommends :) 

https://github.com/koalaman/shellcheck/wiki/SC2086

I understand `shellcheck` is not enforced here, tho. I can remove the quotes if you prefer.

:+1: on exporting the variable